repos/toolset.proj

```diff
@@ -0,0 +1,68 @@
+
+  <PropertyGroup>
+    <BuildCommandArgs Condition="'$(TargetOS)' == 'Linux'">--runtime-id $(TargetRid)</BuildCommandArgs>
+    <BuildCommandArgs>$(BuildCommandArgs) -c $(Configuration)</BuildCommandArgs>
+    <BuildCommandArgs>$(BuildCommandArgs) --stage0 $(DotNetCliToolDir)</BuildCommandArgs>
+
+    <!-- Consume the source-built Core-Setup. This line must be removed to source-build CLI without source-building Core-Setup first. -->
+    <BuildCommandArgs>$(BuildCommandArgs) /p:CoreSetupBlobRootUrl=file:%2F%2F$(LocalBlobStorageRoot)</BuildCommandArgs>
+
+    <!-- Pass in package version props using the Product Construction (ProdCon) API. -->
+    <BuildCommandArgs>$(BuildCommandArgs) /p:PB_PackageVersionPropsUrl=file:%2F%2F$(PackageVersionPropsPath)</BuildCommandArgs>
+
+    <BuildCommandArgs>$(BuildCommandArgs) /p:IncludeNuGetPackageArchive=false</BuildCommandArgs>
+    <BuildCommandArgs>$(BuildCommandArgs) /p:UsePortableLinuxSharedFramework=false</BuildCommandArgs>
+    <BuildCommandArgs>$(BuildCommandArgs) /p:IncludeSharedFrameworksForBackwardsCompatibilityTests=false</BuildCommandArgs>
+
+    <BuildCommand>$(ProjectDirectory)/build$(ShellExtension) $(BuildCommandArgs)</BuildCommand>
+
+    <PackageOutputRid Condition="'$(TargetOS)' == 'Windows_NT'">win-x64</PackageOutputRid>
+    <PackageOutputRid Condition="'$(TargetOS)' == 'OSX'">osx-x64</PackageOutputRid>
+    <PackageOutputRid Condition="'$(PackageOutputRid)' == ''">$(TargetRid)</PackageOutputRid>
+    <ToolsetZipOutput>$(ProjectDirectory)bin/2/$(PackageOutputRid)/packages/</ToolsetZipOutput>
+
+    <RepoApiImplemented>false</RepoApiImplemented>
+    <SourceOverrideRepoApiImplemented>true</SourceOverrideRepoApiImplemented>
+
+    <EnvironmentExternalRestoreSources>$(SourceBuiltPackagesPath)</EnvironmentExternalRestoreSources>
+    <EnvironmentExternalRestoreSources Condition="'$(OfflineBuild)' == 'true'">$(EnvironmentExternalRestoreSources)%3B$(PrebuiltPackagesPath)</EnvironmentExternalRestoreSources>
+
+    <OrchestratedManifestBuildName>N/A</OrchestratedManifestBuildName>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <EnvironmentVariables Include="GitInfoCommitCount=$(GitCommitCount)" />
+    <EnvironmentVariables Include="GitInfoCommitHash=$(GitCommitHash)" />
+
+    <!-- Pass multiple restore sources as environment: avoid the shell seeing ';' in an Exec. -->
+    <EnvironmentVariables Include="ExternalRestoreSources=$(EnvironmentExternalRestoreSources)" />
+
+    <!-- Produce stable versions for RTM only.  When https://github.com/dotnet/source-build/issues/550
+                  is fixed we will be able to determine this automatically.  -->
+    <EnvironmentVariables Include="DropSuffix=true"
+                          Condition="'$(UseStableVersions)' == 'true'" />
+
+    <EnvironmentVariables Include="CLIBUILD_SKIP_TESTS=true" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <RepositoryReference Include="cli" />
```

Comment:
In https://github.com/dotnet/source-build/pull/639, I removed the references to app-insights, fsharp, javascriptservices, netcorecli-fsc, newtonsoft-json, vstest and websdk from repos/cli.proj because it wasn't dependent on these.  These should probably be added to either toolset.proj or core-sdk.proj.

Reply:
I think this might have changed between 2.1 and master - in CLI I see [app-insights](https://github.com/dotnet/cli/blob/ddb64191922a5b5ac75232d71c8209c9f52cadbc/src/dotnet/dotnet.csproj#L58), [newtonsoft](https://github.com/dotnet/cli/blob/bdc5cede292e13f64c648b06b595d7ddb4eca6e5/src/dotnet/dotnet.csproj#L56), and [websdk](https://github.com/dotnet/cli/blob/f0b60ed02c513eb3cdf52e7a6b3353d6b8e52cf8/build/sdks/sdks.csproj#L12).  I added those back to CLI, the other removals still look good.  Also searched for and updated repo references in toolset and core-sdk.